### PR TITLE
Changed from service layers to dynamic layers

### PR DESCRIPTION
**Pull request by gcsantos-gpa**

### CODE REVIEW

File: src/PQDashboard/Scripts/Default.js [hunk 1]
```diff
@@ -3929,35 +3929,22 @@ function loadLeafletMap(theDiv) {
...
-                function () {
-                    return {
-                        color: layer.Color
-                    }
-                };
-        gisLayers[layer.Name] = L.esri.featureLayer(layerProperties);
-    });
-    L.control.layers(null, gisLayers).addTo(leafletMap[currentTab]);
+
+    $.getJSON(baseURI + "/layers?f=pjson&callback=?",
```
Comment (clackner-gpa) [inline src/PQDashboard/Scripts/Default.js:3936]: seems a little strange that you would need  include `?f=jpjson&callback=?` can you double check these are required?
I'd think `/layers?f=pjson` would be sufficent if anything

Comment (gcsantos-gpa) [reply src/PQDashboard/Scripts/Default.js:3936]: It is required. Otherwise, getJSON assumes a regular json and not a jsonp, which gets blocked by CORS

**Review by clackner-gpa — COMMENTED** [review]
I am assuming this is currently deployed at HECCO with this change?

File: src/PQDashboard/Scripts/Default.js (hunk 1 above)
Comment (clackner-gpa) [reply src/PQDashboard/Scripts/Default.js:3936]: You are correct:
https://api.jquery.com/jquery.getjson/
Still seems strange to me but ok will merge.

**Pull request merged**